zndraw/zndraw.py

```diff
@@ -560,3 +569,19 @@ def __getitem__(self, index: int | list | slice) -> ase.Atoms | list[ase.Atoms]:
         if single_item:
             return structures[0]
         return structures
+
+    def insert_local(self, index: int, value: dict):
```

Comment:
you might want to overwrite the `insert` and `append` instead of defining new ones